### PR TITLE
BT-X-7 & BT-X-8

### PR DESCRIPTION
**Pull request by HenriSHS**

### CODE REVIEW

**Comment by stephanstapel** [comment]
@HenriSHS, thanks for the pull request. My initial thought was that adding more parameters to functions such as AddTradeLineItem(). Could you create a function like TradeLineItem.AddLineStatus() that adds the necessary information the trade line item? That would dramatically reduce parameter transfer through the entire component,

Would you mind to add a section to the readme.md to explain when this functionality is needed and how it works?

**Comment by HenriSHS** [comment]
Hope the readme section is fine. Feel free to edit

**Comment by stephanstapel** [comment]
Looks good. Can you backport the functionality and test case to ZUGFeRD 2.0 as well? In ZUGFeRD 1.0, it wasn't available yet.

**Comment by stephanstapel** [comment]
Hi @HenriSHS ,
sme here, is this second pull request ready for review?

File: ZUGFeRD/LineStatusReasonCodes.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        /// For information only
+        /// </summary>
+        INFORMATION
+    }
+
+    internal static class LineStatusReasonCodesExtensions
+    {
+        public static LineStatusReasonCodes? FromString(this LineStatusReasonCodes _, string s)
+        {
+            if (s == null)
```
Comment (stephanstapel) [inline ZUGFeRD/LineStatusReasonCodes.cs:59]: Please use curly brackets for return null;
if (s == null)
{
  return null;
}

**Review by stephanstapel — COMMENTED** [review]
looks really good, thanks. It is not available in UBL, is it?

**Comment by HenriSHS** [comment]
Miss clicked didn't mean t close :D 

> looks really good, thanks. It is not available in UBL, is it?
Unfortunately not

**Comment by stephanstapel** [comment]
Thanks, @HenriSHS !

**Pull request merged**